html-block-and-inline/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Block and Inline</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+</head>
+<body>
+  <h1><b>Blue Americana Snuggie&reg; Blanket</b></h1>
```

Comment:
you don't need the `b` tags in an `h1` tag, the bold text is default styling for all of the `h` elements

Reply:
thank you for pointing that out, will reupload changes shortly.